### ISSUE TITLE
Change working port from 443

### ISSUE THREAD

**Issue by pinionking** [wh1te909/tacticalrmm#293]
Hi,

I already have something running off 443, is there a way to change the main port to something else?

Thanks

**Comment by wh1te909** [comment]
Hi what do you have running on 443? Just throw it behind nginx using virtual hosts, that's what tacticalrmm does. Tactical uses 3 separate backends and that's what nginx is for, to allow multiple services to run on the same server. You can check `/etc/nginx-sites-available/` on a fresh tactical install for examples of how to create a virtual host for your existing service.

**Comment by pinionking** [comment]
Hi,

An Exchange server (on a different subnet) gets 443 for OWA, ActiveSync etc.

I tried running a port redirect for 444 from my firewall to 443, the logon screen loads, but it keeps saying an invalid password. I looked and it is an SSL error, and the connections it is trying to make use the normal 443 port. The calls for API and MESH are without the port numbers so they just hit the exchange server.

I do have a ngnix reverse proxy server to control all the other connections for the other stuff I have running.

**Comment by dinger1986** [comment]
can you run it on a VPS? even just a cheap one?

I don't think mesh allows you to use anything but 443 for communications and that is out with our control.

**Comment by pinionking** [comment]
I worked out a way around it.

I managed to get the Firewall to act as a reverse proxy for https. If it comes in on the right subdomains it goes to Tactical, if not it continues onto the Exchange server.

I had to get the SSL Cert onto the Firewall as well and apply it to those subdomains so it didn't trip out with SSL errors. Just need to remember to update it every three months.

For notes, I am using a Watchguard firewall to do it.
